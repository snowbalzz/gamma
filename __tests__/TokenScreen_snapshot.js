import React from "react";
import { create } from "react-test-renderer";
import TokenScreen from "../app/screens/TokenScreen"

const tree = create(<TokenScreen/>);

test('snapshot', ()=>{
    expect(tree).toMatchSnapshot();
});