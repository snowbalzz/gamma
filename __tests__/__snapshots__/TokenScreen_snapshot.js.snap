// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "teal",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 15,
        "flex": 1,
        "margin": 10,
      }
    }
  >
    <RCTScrollView
      showsHorizontalScrollIndicator={false}
      showsVerticalScrollIndicator={false}
      style={
        Object {
          "marginHorizontal": 20,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "borderRadius": 15,
              "flex": 0.1,
              "flexDirection": "row",
              "margin": 10,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "teal",
                "borderRadius": 15,
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 30,
                  "margin": 10,
                }
              }
            >
               TEST ELEMENT
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
